src/Rx/Weno/facilities.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
this scripts should be moved to interface subdirectory.
Would put it in:
interface/weno/

Reply:
done